CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Basic deserialization tests using different class type specifiers
 - `GammaPrior`, `HalfCauchyPrior`, `NormalPrior`, `HalfNormalPrior`, `LogNormalPrior`
   and `SmoothedBoxPrior` can now be chosen as lengthscale prior
+- User guide about environment variables for configuration
```

Comment:
Aren't environment variables essentially always "for configuration"?
```suggestion
- Environment variables user guide
```

Reply:
technically they can also be for other things like storing secrets or so, but i can change the title nonetheless